### PR TITLE
only show youtube atom duration if not on front

### PR DESCRIPTION
**Pull request by GHaberis**
## What does this change?

On fronts we don't want to show the bottom bar with the icon/duration

## What is the value of this and can you measure success?

N/A

## Does this affect other platforms - Amp, Apps, etc?

No

## Screenshots

Before...

picture 29

and after...

picture 28

## Tested in CODE?

No

@gidsg @duarte

### CODE REVIEW

**Comment by duarte** [comment]
Looks good to me re: timestamp!

Can we get the button to go a size bigger on large cards — or should that go in a separate PR?

**Comment by TBonnin** [comment]
I am curious as a reader: what lead to this decision to hide the duration?
As a reader I find it useful to know how long a video is gonna be. That's might influence me to click (or not click) on it

File: static/src/javascripts-legacy/projects/common/modules/atoms/youtube.js [hunk 1]
```diff
@@ -115,9 +115,9 @@ define([
         };
 
         if (overlay) {
-            var formattedDuration = getFormattedDuration(players[atomId].player.getDuration());
-            
-            setDuration(formattedDuration, overlay);
+            if (!config.page.isFront) {
```
Comment (gidsg) [inline static/src/javascripts-legacy/projects/common/modules/atoms/youtube.js]: We could work this out server side and pass it down as a parameter on the template.

**Comment by duarte** [comment]
@TBonnin removing it from playable videos on fronts is for parity with the current (non-youtube) video setup. In small cards, and with the in-article player timestamp, the image area becomes too busy in small cards. We have alternate designs for timestamp, but they have so far tested to reduce video plays — so that still needs work

**Comment by gidsg** [comment]
LGTM, but could do with another pair of eyes as I've worked on this...

**Review by akash1810 — APPROVED** [review]
lgtm

**Pull request merged**